server/elasticsearch/src/main/resources/hawkular-apm-mapping.json

```diff
@@ -17,11 +17,6 @@
             }],
             "date_detection" : false,
             "numeric_detection": false,
-            "_timestamp": {
```

Comment:
Can you create a separate jira for this, as I think it can be addressed by using a custom serializer to essentially store an internal field in ES which could be referenced by the index mapping. As otherwise the timestamps won't be accurate.